src/app/Dashboard/styles/dashboard.scss

```diff
@@ -78,15 +78,14 @@
   .button__add {
     height: rem(32px);
     width: rem(143px);
-    background-color: #F5A623;
-    box-shadow: inset 0 1px 3px 0 rgba(0,0,0,0.5), 0 12px 35px 0 rgba(52,77,178,0.34);
+    background-color: $orange;
```

Comment:
Do we really have orange in the brand guideline?

Reply:
No, we do not. I have changed it to Andela Gold.